includes/class-wc-post-data.php

```diff
@@ -254,7 +254,10 @@ public static function wp_insert_post_data( $data ) {
 					$data['post_parent'] = 0;
 				break;
 			}
+		} elseif ( 'product' === $data['post_type'] && 'auto-draft' === $data['post_status'] ) {
```

Comment:
Not really, since this is created without any `$_POST`, it's created new you click in `Add new product` by an URL.